pom.xml

```diff
@@ -568,7 +568,7 @@
             <name>Open Source Geospatial Foundation Repository</name>
             <url>http://download.osgeo.org/webdav/geotools/</url>
```

Comment:
@JLLeitschuh The commented out url two lines below was picked up while the actually used url stayed http-only. Maybe the automatic redirect to https by the website itself threw the parser off?

Reply:
Odd. I don't actually try to run any of the URLs.

This was using a python regular expression to figure out what lines to replace.

This is an XML configuration I 100% didn't expect.

Here's the regular expression I used.

https://github.com/JLLeitschuh/bulk-security-pr-generator/blob/59250784e68339b3ea7aa47faf6f281cdf47afb9/do_security_fix.py#L36-L40